ApiaryReportingApi.md

```diff
@@ -213,14 +241,15 @@ HOST: http://api.apiary.io/
         ```
 
 
-## Group Test Steps 
+## Group Test Steps
 
 - Attributes
   - _id (string) ... Unique object id. DO NOT USE IT WHEN CREATING! Used in post body only for purpose of testing this blueprint.
   - testRunId (string) ... Reference to the parent test run identificator
   - origin (object) ... Origin path (position, location) of the step in the blueprint AST
+    - filename (string) .. File name of original blueprint
```

Comment:
You used `file` key in `test-run` definition . Please make these two choices the same.